server/controllers/stock/reports/index.js

```diff
@@ -663,6 +689,16 @@ function stockMovementsReport(req, res, next) {
       data.csv = rows;
       data.display = display;
 
+      // group by depot
+      let depots = _.groupBy(rows, d => d.depot_text);
```

Comment:
At some point, we will likely have move this into SQL for performance reasons.  It would be good to put a `// TODO` here so that we can easily locate it.